### PR TITLE
Updated the HOSS Gitspaces Docs

### PR DESCRIPTION
**Pull request by khushisharmaharness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/open-source/gitspaces/develop-in-gitspaces/forward-ports.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-## Explain about the port forwarding principles we are using
-
 You can forward ports in your Gitspaces to test and debug your application. 
 
 Port forwarding gives you access to TCP ports running within your Gitspace. For example, if you're running a web application on a particular port in your Gitspace, you can forward that port. This allows you to access the application from the browser on your local machine for testing and debugging.
 
 When an application running inside a Gitspace prints output to the terminal that contains a localhost URL, such as `http://localhost:PORT` or `http://127.0.0.1:PORT`, the port is automatically forwarded. If you're using Gitspace in the browser or in Visual Studio Code, the URL string in the terminal is converted into a link that you can click to view the web page on your local machine.
 
-## How to use port forwarding
+## How to use Port Forwarding
```
Comment (SushrutHarness) [inline docs/open-source/gitspaces/develop-in-gitspaces/forward-ports.md:14]: What style guide are we following for headers? This one or the HDH one mentioned here: https://harness.atlassian.net/wiki/spaces/DOCS/pages/21544468526/Harness+documentation+style+guide#Headings Specifically " Use sentence style capitalization"

**Review by wei-harness — APPROVED** [review]
tsx and json files look good to me

**Comment by dewan-ahmed** [comment]
@khushisharmaharness as suggested on Slack, pls add redirects for any deleted page(s)

> I see that you deleted a page. Please add a redirect: https://harness.atlassian.net/wiki/spaces/HDH/pages/21469528316/HDH+Front+Matter+Re-direct

File: docs/open-source/gitspaces/develop-in-gitspaces/forward-ports.md (hunk 1 above)
Comment (khushisharmaharness) [reply docs/open-source/gitspaces/develop-in-gitspaces/forward-ports.md]: @SushrutHarness 
Fixed this. Thanks for the feedback.

**Comment by khushisharmaharness** [comment]
@dewan-ahmed Please approve this. (Added the redirect)

**Review by rohanmaharjan100 — DISMISSED** [review]

**Review by dewan-ahmed — APPROVED** [review]
LGTM

**Review by rustd — APPROVED** [review]

**Pull request merged**